content/en/docs/tasks/debug-application-cluster/debug-service.md

```diff
@@ -61,8 +61,7 @@ kubectl scale deployment hostnames --replicas=3
 deployment.apps/hostnames scaled
 ```
 
-{{< note >}}
-This is the same as if you had started the Deployment with the following
+Note that this is the same as if you had started the Deployment with the following
```

Comment:
For clarity (and to avoid future changes making this back into a note), I'd write:
```suggestion
This is the same as if you had started the Deployment with the following
```